package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "name-request",
-  "version": "2.1.5",
+  "version": "2.1.6",
```

Comment:
This is OK, or we could consider it a "significant change" and set this to 2.2.0 instead. Thoughts?

Reply:
It's a full feature for sure, i think `2.2.0` makes sense!